core.yml

```diff
@@ -0,0 +1,314 @@
+# See documentation for details on definitions:
+# https://rubocop.readthedocs.io
+
+require:
+  - rubocop-performance
+  - rubocop-rails
```

Comment:
don't we only want to require `rubocop-rails` for the `rails.yml` configuration?

Reply:
Good catch